recipes/libcanberra/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "libcanberra" %}
+{% set version = "0.30" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://0pointer.de/lennart/projects/libcanberra/libcanberra-{{ version }}.tar.xz
+  sha256: c2b671e67e0c288a69fc33dc1b6f1b534d07882c2aceed37004bf48c601afa72
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - libvorbis
+    - libtool
+  run:
+    - libvorbis >=1
+    - libtool >=2
```

Comment:
This should probably be pinned to major version like,
`- {{ pin_compatible("libtool") }}`